### DISCUSSION TITLE
Where to search package of termux?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [termux/termux-packages#16727] [Q&A] [answered]
For other distribution, it usually have a website to search the package like <https://packages.msys2.org/>, <https://search.nixos.org/packages>, <https://archlinux.org/packages/>. Where to search package of termux?

**Comment by agnostic-apollo** [comment, marked as answer]
https://repology.org/repository/termux

https://github.com/termux/repology-metadata

**Comment by tareksander** [comment]
I made a simple website for that that is a bit easier to use than repology IMO: https://tareksander.github.io/termux-package-search/

**Comment by twaik** [comment]
Also you can search packages related to files using `apt-file search <somefile>`.
